sycl/test-e2e/KernelAndProgram/free_function_kernels.cpp

```diff
@@ -212,7 +212,7 @@ SYCL_EXTERNAL SYCL_EXT_ONEAPI_FUNCTION_PROPERTY((
   ptr2D[GId.get(0)][GId.get(1)] = LId.get(0) + LId.get(1) + start;
 }
 
-// Explicit instantiation with �int*�.
+// Explicit instantiation with �int*�.
```

Comment:
`ï¿½int*ï¿½`?

Reply:
Something weird is happening here like github bug or something, as every time I push this it appears in a special weird character that I don't even touch it.

Reply:
I transformed it from `”` to " character.